src/lib/variants.test.ts

```diff
@@ -80,196 +80,212 @@ const targetingDefault: EpicTargeting = {
     mvtId: 2,
 };
 
+describe('find variant', () => {
```

Comment:
I think this is a carry over from working with Ruby/RSpec lots over the years, but I really like it when the `describe` block has the name of the function or method under test. What do you think?

```suggestion
describe('findTestAndVariant`, () => {
```

Reply:
Cool, thanks @tjmw. Me too, I think I just missed this one so have updated now.
Btw, for the filter tests below (the majority of test blocks on that file) I'm using the filter name + the word "filter" so when you read the sentence it's a bit more clear that we're testing filters, which always return a boolean value (true/false or pass/fail).